.env.example

```diff
@@ -0,0 +1,6 @@
+WHO_AM_I_TOKEN=
+JWT_FORGED_TOKEN=
```

Comment:
There is 3 forged jwt token env variables in here. Think of a more fitting name.